app/modules/conversations/conversations_router.py

```
@@ -566,6 +577,119 @@ async def resume_session(
+                "total": total_tokens,
+            },
+            "remaining": remaining,
+            "usage_percentage": round(usage_percentage, 2),
+            "warning_level": warning_level,
+        }
+
+    except HTTPException:
+        raise
+    except Exception as e:
+        logger.error(f"Error getting context usage: {e}", exc_info=True)
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)